samples/net/sockets/packet/README.rst

```diff
@@ -0,0 +1,28 @@
+.. _packet-socket-sample:
+
+Raw socket sample
+#################
+
+Overview
+********
+
+This sample is a simple packet socket application showing usage
+of packet socket over ethernet. Sample prints every packet which
```

Comment:
``Ethernet`` is a trademarked name, so needs to be upper case.

Here's a rewrite of this paragraph with some wording edits:
```
This sample is a simple packet socket application showing usage
of packet sockets over Ethernet. The sample prints every packet
received, and sends a dummy packet every 5 seconds.
The Zephyr network subsystem does not touch any of the headers
(L2, L3, etc.). 
```

Reply:
Comments are addressed in original PR https://github.com/zephyrproject-rtos/zephyr/pull/12564